Bottom: 5a80a577ab16ea337ac8224226e1fda9436a0e94
Top:    2e283ed1478517f64389d816b770213efa34ea98
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 16:31:26 +0000

Disable debug


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index a324066..cfd5d6a 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1221,7 +1221,7 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
                               copper_appearance,
                               NULL));
 
-    break;
+    //break;
   }
 
 
@@ -1233,10 +1233,10 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
 //  gui->invalidate_all ();
 
   /* ASSUME THERE IS A POLYGON WHERE WE KNOW WE PUT ONE... */
-  ((PolygonType *)PCB->Data->Layer[1].Polygon->data)->Clipped = info.poly;
-  gui->invalidate_all ();
+//  ((PolygonType *)PCB->Data->Layer[1].Polygon->data)->Clipped = info.poly;
+//  gui->invalidate_all ();
 
-//  poly_Free (&info.poly);
+  poly_Free (&info.poly);
 
   return objects;
 }
